CONTRIBUTING.md

```diff
@@ -38,6 +38,16 @@ application.
 When you make further changes to the extension, quit the "Extension Development
 Host" and run `F5` (or `Run > Start debugging`) again.
 
+## Publishing and testing locally
+
+To ensure that everything is working as expected after changes it's best to
+publish a vsix artifact locally. You can do this by `yarn build` which will result in
+the metals-<version>.vsix being built and placed in the root of your directory.
+You can then simply `code <path-to-vsix-file>` which will install the extension.
```

Comment:
does it work this way? I remember that was a flag for it like —install or something

Reply:
Yea, you can actually just open up a vsix file via the file opener as well and it will install it. 
> Install a Packaged Extension (.vsix)

>You can manually install an VS Code extension packaged in a .vsix file. Simply install using the VS Code command line providing the path to the .vsix file.

>`code myExtensionFolder\myExtension.vsix`

>The extension will be installed under your user .vscode/extensions folder. You may provide multiple .vsix files on the command line to install multiple extensions at once.

>You can also install a .vsix by opening the file from within VS Code. Run File | Open File... or kb(workbench.action.files.openFile) and select the extension .vsix.

Reply:
Ah, TIL! Thanks